lab_1/main.py

```diff
@@ -105,7 +224,14 @@ def write_to_file(path_to_file: str, content: list):
     """
     Writes the result in a file
     """
-    pass
+    text = ''
+    file = open (path_to_file, 'w')
+    for small_list in content:
+        for word in small_list:
+            text += word + ' '
+        text += '\n'
+        file.write (text)
```

Comment:
put all joined lines into a variable
only then write it to file

it is not good to write to file at each iteration